.github/workflows/manual-benchmark.yaml

```diff
@@ -0,0 +1,35 @@
+name: Manual Benchmark
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+    inputs:
+        qdrant_branch:
+            description: "Branch of qdrant to benchmark"
+            default: dev
+        datasets:
+            description: "Dataset to benchmark"
+            default: laion-small-clip
+        benchmark_table:
+            description: "Postgres table to store benchmark results"
+            default: benchmark_manual
```

Comment:
This will show equivalent input boxes before running the workflow.

Reply:
Will add an input for `datasets` after we test that this PR is working as expected.